packages/shared/src/modules/onboarding/libs/enums/onboarding-validation-rule.enum.ts

```diff
@@ -0,0 +1,5 @@
+const OnboardingValidationRule = {
+	QUESTIONS_LENGTH: 6,
```

Comment:
This will create issues if we decide to add new questions in future. Let's have this check in Backend